content/en/blog/2023/why-histograms/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Why Histograms?
+date: 2023-05-05
+author: '[Daniel Dyla](https://github.com/dyladan)'
+---
+
+<i>Originally posted to
+https://dyladan.me/histograms/2023/05/02/why-histograms/</i>
```

Comment:
```suggestion
canonical_url: https://dyladan.me/histograms/2023/05/02/why-histograms/
---

```

Reply:
To be consistent with our other posts, please add the following at the end of the page:

```markdown
_A version of this article was [originally posted][] to the author's blog._

[originally posted]: {{% param canonical_url %}}
```